### PR TITLE
Laboratory work #8, Darya Bugakova - 20FPL1

### PR DESCRIPTION
**Pull request by daribbb**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+datasets==2.16.1
```
Comment (SalnikovIgor) [inline requirements.txt:1]: Add line  `--extra-index-url https://download.pytorch.org/whl/cpu`  if `torch` is needed

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Fix past comments

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Fix past comments

File: lab_7_llm/main.py [hunk 2]
```diff
@@ -6,19 +6,11 @@
 from pathlib import Path
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: Checkout the files from 7 lab to the master

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_8_llm/main.py [hunk 3]
```diff
@@ -32,7 +23,7 @@
 
 class RawDataImporter(AbstractRawDataImporter):
     """
-    A class that imports the HuggingFace dataset.
+    A class that imports the HuggingFace dataset..
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: ..?

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_8_llm/main.py (hunk 3 above)
Comment (daribbb) [reply lab_8_llm/main.py]: fixed this

**Pull request closed without merging**